src/styles/blocks/_nav.scss

```diff
@@ -0,0 +1,46 @@
+.nav {
+  &__list {
+  display: flex;
+}
+
+  &__item {
+  margin-right: 20px;
+}
+
+  &__item:last-child {
+  margin-right: 0;
+}
+
+  &__link {
+  display: block;
+  line-height: 60px;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+  color: black;
+
+  transition: color 0.2s;
+}
+
+  &__link:hover {
+  color: #00acdc;
+ }
+}
+.is-active {
+  position: relative;
+  color: #00acdc;
+
+  &:before {
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+
+  height: 4px;
+  border-radius: 8px;
+
+  background-color: #00acdc;
```

Comment:
let's create a variable for this color as well, you use it a couple of times across your project